scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1560,6 +1560,13 @@
                (counters/dec! in-order-queue-counter)
                (body-fn))))))
 
+(defn prepare-match-trigger-chan
+  "Calculate the required interval for match-trigger-chan and start the chimes"
+  [match-trigger-chan pools]
+  (let [{:keys [per-pool-match-interval-millis global-min-match-interval-millis]} (config/matching-settings)
+        match-interval-millis (max (int (/ per-pool-match-interval-millis (count pools))) global-min-match-interval-millis)]
```

Comment:
Thoughts on using `->` here?

```suggestion
        match-interval-millis (-> per-pool-match-interval-millis 
                                  (/ (count pools)) 
                                  int 
                                  (max global-min-match-interval-millis))]
```

Reply:
Seems like we should log a warning-level message if `(= match-interval-millis global-min-match-interval-millis)` -- basically "heads up, you're being throttled".